frontend/src/metabase/css/core/base.styled.ts

```diff
@@ -0,0 +1,89 @@
+import type { Theme } from "@emotion/react";
+import { css } from "@emotion/react";
+
+export const getRootStyle = (theme: Theme) => css`
```

Comment:
I split stuff moved from `base.modules.css` into 2 parts.
1. Root style -> this applies to `body` for app, and the root container of the React SDK
2. base style -> this doesn't change on the app, but is scope under the root container of the React SDK

The reason I need to split is because stuff in root style should be defined once. Previously CSS custom properties are defined in `:root` this doesn't work when in SDK, so I combined both CSS custom properties from `:root` and other global styles from `body` into a single scope. This works great because:
- in the app: `body` will contain everything, so it also acts like a `:root` in that way.
- in SDK: We apply all these root styles to the root container of the SDK, so these styles are only scoped to anything under this root div. So the style won't leak to user apps.